static/js/components/PersonalTab.js

```diff
@@ -58,16 +60,29 @@ class PersonalTab extends React.Component {
     setProgram(selected);
   }
 
+  getProgramId = (selectedProgram: Program, currentProgramEnrollment: ProgramEnrollment) => {
```

Comment:
to get flow typing to recognize the return type, you could define the function like this:

`getProgramId = (selectedProgram: Program, currentProgramEnrollment: ProgramEnrollment): number|null => {`